pagopa/io-app
# fix(IDPay): [IODPAY-193] Missing I18n key for network error in initiative details screen

## Short description
added a check for the correct datatype in `actions.ts`, which shows a toast in `InitiativeDetailsScreen`

## List of changes proposed in this pull request
- added a check for the correct datatype, where if the error does not match it, a generic one will be shown

## How to test

from an initiative details' page, navigate to `instruments/IBAN EnrollmentScreen` and force it to throw an error, by modifying the machine.
the correct message that will be shown in case of a network error should be along the lines of "there was a problem, try again"
